### PR TITLE
FAME file storage & VM backup detectors

### PR DESCRIPTION
**Pull request by BzSpi**

### CODE REVIEW

File: modules/fame_azure-storage-file-backup/variables-gen.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+variable "backup_notifications" {
+  description = "Notification recipients list per severity overridden for backup detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "backup_aggregation_function" {
+  description = "Aggregation function and group by for backup detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
```
Comment (Shr3ps) [inline modules/fame_azure-storage-file-backup/variables-gen.tf:12]: I don't remember well, empty means aggregate by all available dimensions?

File: modules/fame_azure-vm-backup/tags.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+locals {
+  tags = ["fame", "azure-storage-file-backup"]
```
Comment (Shr3ps) [inline modules/fame_azure-vm-backup/tags.tf:2]: wrong copy/paste

File: modules/fame_azure-vm-backup/conf/01-vm-backup.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module: Azure VM backup
+name: success
+id: backup
```
Comment (Shr3ps) [inline modules/fame_azure-vm-backup/conf/01-vm-backup.yaml:3]:
```suggestion
id: vm backup
```

File: modules/fame_azure-storage-file-backup/conf/01-file-backup.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+module: Azure Storage File backup
+name: success
+id: backup
```
Comment (Shr3ps) [inline modules/fame_azure-storage-file-backup/conf/01-file-backup.yaml:3]:
```suggestion
id: file backup
```

File: modules/fame_azure-vm-backup/tags.tf (hunk 2 above)
Comment (Shr3ps) [reply modules/fame_azure-vm-backup/tags.tf:2]:
```suggestion
  tags = ["fame", "azure-vm-backup"]
```

**Review by Shr3ps — CHANGES_REQUESTED** [review]

File: modules/fame_azure-vm-backup/conf/01-vm-backup.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (Shr3ps) [inline modules/fame_azure-vm-backup/conf/01-vm-backup.yaml:4]: you may want to add an explicit:

```
transformation: true
aggregation: true 
```

File: modules/fame_azure-storage-file-backup/variables-gen.tf (hunk 1 above)
Comment (BzSpi) [reply modules/fame_azure-storage-file-backup/variables-gen.tf:12]: That's right

**Review by Shr3ps — APPROVED** [review]

**Review by Shr3ps — APPROVED** [review]

**Pull request merged**